src/uucore/src/lib/features/sum.rs

```diff
@@ -319,7 +362,7 @@ macro_rules! impl_digest_common {
 
 // Implements the Digest trait for sha2 / sha3 algorithms with variable output
 macro_rules! impl_digest_shake {
-    ($algo_type: ty) => {
+    ($algo_type: ty, $output_bits: literal) => {
```

Comment:
What's the reason that the definition here is different than the one used for `impl_digest_common` (`literal` vs `expr`)?

Reply:
I would use `literal` in both cases. In the other macro the number is also written directly.